src/main/java/fr/epicanard/globalmarketchest/auctions/AuctionInfo.java

```diff
@@ -85,6 +85,34 @@ public Double getTotalPrice() {
     return BigDecimal.valueOf(this.price).multiply(BigDecimal.valueOf(this.amount)).doubleValue();
   }
 
+  /**
+   * Get final total price of auction after tax
+   *
+   * @return Final total price after tax
+   */
+  public Double getTaxedPrice() {
+    //Check if the tax is a valid multiplier or bad things may happen if configured outside the proper bounds
+    double taxMultiplier = 1 - ConfigUtils.getDouble("Options.ServerTax", 0);
+    if (taxMultiplier < 0 || taxMultiplier > 1) {
+      taxMultiplier = 0.0;
+    }
+    return BigDecimal.valueOf(getTotalPrice()).multiply(BigDecimal.valueOf(taxMultiplier)).doubleValue();
```

Comment:
A tax isn't a price that you add to existing price ?

So maybe this formula is more logic : `totalPrice + (totalPrice * taxMultiplier)`

You can also re-use **getTaxAmount**.
```suggestion
    return BigDecimal.valueOf(getTotalPrice()).add(BigDecimal.valueOf(getTaxAmount())).doubleValue();
```

Reply:
Oh no, I went with the other tax option. When someone creates an offer, he specifies the amount other players will pay, so it's like a price with VAT incluided. Then when the trade occurs the buyer doesn't get a more expensive price, money just get deducted from the price and the seller receives the original listing price - 20%

I understand what you thought taxes are and that's another perfectly valid way to approach it, I chose this one because most plugins with taxes are using it and users are usually already familiar with it.

Reply:
And thats why the formula is what it is = (1 - multiplier) * original price, but you are totally right, I could have deducted it since I already calculated it once.

Reply:
Hum ok strange way of thinking. So it's not a "TaxedPrice" but more like a "PriceWithoutTax"